Makefile.include

```diff
@@ -413,8 +421,6 @@ endif
 # Assume GCC/GNU as supported toolchain if CPU's Makefile.include doesn't
 # provide this macro
 TOOLCHAINS_SUPPORTED ?= gnu
-# Import all toolchain settings
-include $(RIOTMAKE)/toolchain/$(TOOLCHAIN).inc.mk
```

Comment:
Isn't this is the only thing that needs to be moved?

Reply:
no we need acces to the specifig toolchain therefor the cpu prefix definition has to move up